plugins/eddn.py

```diff
@@ -424,51 +570,64 @@ def filter_localised(d):
         # add mandatory StarSystem, StarPos and SystemAddress properties to Scan events
         if 'StarSystem' not in entry:
             if not system:
-                return("system is None, can't add StarSystem")
+                return "system is None, can't add StarSystem"
```

Comment:
Add logger.debug(...) for these cases ?

Reply:
Do we want to change behavior there? that return ends up in [EDMarketConnector.py](https://github.com/EDCD/EDMarketConnector/blob/develop/EDMarketConnector.py#L671-L680) and does some sound silliness

Reply:
Keep the return.  My thinking is that this looks like a "should never happen" thing so needs logging.  The 'sound silliness' alerts the user who might then actually send us logs.

Reply:
possibly a warn then over a debug?

Reply:
Sure.